### ISSUE TITLE
SCSS lint error

### ISSUE THREAD

**Issue by jmbelloteau** [stylelint/stylelint#2472]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I'm having issue with **stylelint** and the **stylelintfmt** plugin with a SCSS function.
A space is added with **stylelint**  config where is shouldn't.

> Which rule, if any, is this issue related to?

`at-rule or selector expected`

> What CSS is needed to reproduce this issue?

```scss
@for $i from 1 through length ($breakpoints) {
    // some code
  }
```
The error disappear when I remove the space between **length** and **($breakpoints)**:

```scss
@for $i from 1 through length($breakpoints) {
    // some code
  }
```
> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard",
  "plugins": [
    "stylelint-scss",
    "stylelint-order"
  ],
  "rules": {
    "color-hex-case": "lower",
    "color-hex-length": "long",
    "number-leading-zero": "never",
    "value-list-max-empty-lines": 1,
    "comment-empty-line-before": "never",
		"order/order": [
      "custom-properties",
			"declarations"
		]
  }
}
```

> Which version of stylelint are you using?

"stylelint": "^7.9.0",

> How are you running stylelint: CLI, PostCSS plugin, Node API?

With stylelint-webpack-plugin.

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SCSS maps.

> What did you expect to happen?

I need to know how to disable the concerned rule or even better create an exception for this case.

**Comment by hudochenkov** [comment]
Please use issue template to describe your problem.

**Comment by hudochenkov** [comment]
SCSS compiler throw error if there is a space after `length`. [Try it](http://www.sassmeister.com/gist/d83fe58792aacdb15036db3917faa489).

Until issue isn't using our issue template, there is nothing we could do.

**Comment by jmbelloteau** [comment]
Hi @hudochenkov,

Thanks for you answer but I already talk about this on my post...
Please tell me witch informations you are missing?

(I've follow your template... sorry if something is missing, I don't understand)

**Comment by evilebottnawi** [comment]
@jmbelloteau `stylelint` don't validate `css` (`scss`, `less` and etc) code

**Comment by jmbelloteau** [comment]
@evilebottnawi @hudochenkov 

I simply need to know witch **stylelint** rule is telling that a space is needed between **length** and **($breakpoints)**.

And how create a new rule to tell  **stylelint** to make an execption or disable it.

**Comment by evilebottnawi** [comment]
@jmbelloteau you can create issue about scss syntax into https://github.com/kristerkari/stylelint-scss, also your can create own rule, use https://stylelint.io/developer-guide/ documentation for this.

**Comment by hudochenkov** [comment]
You didn't provide and error message and stylelint version. Error message contains rule name.

**Comment by jmbelloteau** [comment]
@hudochenkov oh ok sorry I've just update the first post with this informations

**Comment by evilebottnawi** [comment]
@jmbelloteau see my comment https://github.com/stylelint/stylelint/issues/2472#issuecomment-292142211

**Comment by jmbelloteau** [comment]
@evilebottnawi ok thanks

**Comment by hudochenkov** [comment]
@jmbelloteau thank you, but there is still no full error message :(

**Comment by jeddy3** [comment]
I'm unable to reproduce this with `7.10.1`.

The following config:

```json
{
  "extends": "stylelint-config-standard",
  "plugins": [
    "stylelint-scss",
    "stylelint-order"
  ],
  "rules": {
    "color-hex-case": "lower",
    "color-hex-length": "long",
    "number-leading-zero": "never",
    "value-list-max-empty-lines": 1,
    "comment-empty-line-before": "never",
		"order/order": [
      "custom-properties",
			"declarations"
		]
  }
}
```

Produces _no errors_ for _neither_ of the following:

```scss
@for $i from 1 through length ($breakpoints) {
  // some code
}
```

```scss
@for $i from 1 through length($breakpoints) {
  // some code
}
```

I'm going to close this issue.

@jmbelloteau Please ensure you are running `7.10.1`. If this doesn't fix the error, feel free to create a new issue, but be sure to include the "What actually happened (e.g. what warnings or errors you are getting)?" part of the template. We need to information to diagnose the problem.

**Comment by jimblue** [comment]
Hi @jeddy3  @hudochenkov @evilebottnawi   !

Has you can see I've change my Github user name for jimblue. Keeping jmbelloteau was not good for SEO as my website is jmbelloteau.com and I started to have many post on Github.

Because Github doesn't do it automatically could you please guy change the reference to @jmbelloteau for @jimblue in your post above?

Thank you for helping me with that.

I'll remove this message once it's done!

Cheers

**Comment by jeddy3** [comment]
Done
